Fixing tests

**Describe the bug**
We have a few issues with the tests, causing the builds to fail.

**To Reproduce**
Taking a look at the build logs, we see the following errors:


*Ubuntu*
```
======================================================================
FAIL: test_download_mandates_csv (test_module.TestScholarly)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/runner/work/scholarly/scholarly/test_module.py", line [66](https://github.com/scholarly-python-package/scholarly/runs/6168033939?check_suite_focus=true#step:7:66)8, in test_download_mandates_csv
    self.assertEqual(policy[agency_index], agency_policy[agency])
AssertionError: '82%' != ''
- 82%
+ 


======================================================================
FAIL: test_related_articles_from_author (test_module.TestScholarly)
Test that we obtain related articles to an article from an author
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/runner/work/scholarly/scholarly/test_module.py", line 578, in test_related_articles_from_author
    self.assertEqual(pub[key], same_article[key])
AssertionError: [71](https://github.com/scholarly-python-package/scholarly/runs/6168033939?check_suite_focus=true#step:7:71)856 != 718[82](https://github.com/scholarly-python-package/scholarly/runs/6168033939?check_suite_focus=true#step:7:82)

----------------------------------------------------------------------
```

*Mac*
```
FAIL: test_download_mandates_csv (test_module.TestScholarly)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/runner/work/scholarly/scholarly/test_module.py", line [66](https://github.com/scholarly-python-package/scholarly/runs/6168034015?check_suite_focus=true#step:7:66)8, in test_download_mandates_csv
    self.assertEqual(policy[agency_index], agency_policy[agency])
AssertionError: '82%' != ''
- 82%
+

======================================================================
FAIL: test_related_articles_from_author (test_module.TestScholarly)
Test that we obtain related articles to an article from an author
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/runner/work/scholarly/scholarly/test_module.py", line 5[78](https://github.com/scholarly-python-package/scholarly/runs/6168034015?check_suite_focus=true#step:7:78), in test_related_articles_from_author
    self.assertEqual(pub[key], same_article[key])
AssertionError: 71856 != 718[82](https://github.com/scholarly-python-package/scholarly/runs/6168034015?check_suite_focus=true#step:7:82)

======================================================================
FAIL: test_related_articles_from_publication (test_module.TestScholarly)
Test that we obtain related articles to an article from a search
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/runner/work/scholarly/scholarly/test_module.py", line 605, in test_related_articles_from_publication
    self.assertEqual(related_article['bib']['title'], 'Large Magellanic Cloud Cepheid standards provide '
AssertionError: 'Planck 2015 results-xiii. cosmological parameters' != 'Large Magellanic Cloud Cepheid standards [110 chars]ΛCDM'
- Planck 2015 results-xiii. cosmological parameters
+ Large Magellanic Cloud Cepheid standards provide a 1% foundation for the determination of the Hubble constant and stronger evidence for physics beyond ΛCDM
```


*Windows*
```
======================================================================
ERROR: test_download_mandates_csv (test_module.TestScholarly)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "D:\a\scholarly\scholarly\test_module.py", line [66](https://github.com/scholarly-python-package/scholarly/runs/6168034099?check_suite_focus=true#step:7:66)7, in test_download_mandates_csv
    agency_index = funder.index(agency)
ValueError: 'US National Science Foundation' is not in list

======================================================================
ERROR: test_search_author_single_author (test_module.TestScholarly)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "D:\a\scholarly\scholarly\test_module.py", line 305, in test_search_author_single_author
    scholarly.pprint(author)
  File "D:\a\scholarly\scholarly\scholarly\_scholarly.py", line 437, in pprint
    print(pprint.pformat(to_print))
  File "c:\hostedtoolcache\windows\python\3.8.10\x64\lib\encodings\cp1252.py", line 19, in encode
    return codecs.charmap_encode(input,self.errors,encoding_table)[0]
UnicodeEncodeError: 'charmap' codec can't encode character '\u015f' in position 6[68](https://github.com/scholarly-python-package/scholarly/runs/6168034099?check_suite_focus=true#step:7:68)1: character maps to <undefined>

======================================================================
FAIL: test_related_articles_from_author (test_module.TestScholarly)
Test that we obtain related articles to an article from an author
----------------------------------------------------------------------
Traceback (most recent call last):
  File "D:\a\scholarly\scholarly\test_module.py", line 578, in test_related_articles_from_author
    self.assertEqual(pub[key], same_article[key])
AssertionError: [71](https://github.com/scholarly-python-package/scholarly/runs/6168034099?check_suite_focus=true#step:7:71)856 != 718[82](https://github.com/scholarly-python-package/scholarly/runs/6168034099?check_suite_focus=true#step:7:82)

----------------------------------------------------------------------

```

It is a bit confusing what causes the issues, especially the issues associated with the mandates calls

--- a/test_module.py
+++ b/test_module.py
@@ -655,8 +655,8 @@ class TestScholarly(unittest.TestCase):
                 "Swedish Research Council for Environment, Agricultural Sciences and Spatial Planning": ""
             }
             agency_2020 = {
-                "US National Science Foundation": "86%",
-                "Department of Science & Technology, India": "48%",
+                "US National Science Foundation": "87%",
+                "Department of Science & Technology, India": "49%",
                 "Swedish Research Council": "89%",
                 "Swedish Research Council for Environment, Agricultural Sciences and Spatial Planning": "88%"
             }

--- a/test_module.py
+++ b/test_module.py
@@ -533,7 +533,7 @@ class TestScholarly(unittest.TestCase):
         """
         author = scholarly.search_author_id("f4KlrXIAAAAJ")
         scholarly.fill(author, sections=['basics', 'public_access', 'publications'])
-        self.assertGreaterEqual(author["public_access"]["available"], 1180)
+        self.assertGreaterEqual(author["public_access"]["available"], 1150)
         self.assertEqual(author["public_access"]["available"],
                          sum(pub.get("public_access", None) is True for pub in author["publications"]))
         self.assertEqual(author["public_access"]["not_available"],

